tests/install_go.sh

```diff
@@ -9,6 +9,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+set -x
```

Comment:
Is it really useful all the time or should it be gated by a `[[ -n "${DEBUG}" ]] &&` ?